lib/ama/styles/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module AMA
   module Styles
-    VERSION = '3.23.0'
+    VERSION = '3.23.1'
```

Comment:
what do you think about using 3.24.0 since it's really more of a new feature than a hotfix?